runbook.md

```diff
@@ -0,0 +1,17 @@
+# Runbook for proposed changes with steps
+
+## -- changes:
+ 1. secure access to prod trust claim DB
+ 2. change digital ocean droplet to test server with a separate DB for trust claim
+ 3. maintain digital ocean for Taiga and Aws for trust claim
+ 4. set-up CI/CD on dev (digital ocean trustclaim server)
+ 5. rewrite taiga.whatscookin.us to redirect to Taiga.linkedtrust.us
+    
+## -- steps:
+ 1. for secure access to prod DB, I'll only allow the Aws prod IP to be the only inbound traffic IP
+2. for this, I'll enable the DNS name 'lives.linkedtrust.us' to be the dev domain and the IP being the dev IP, which is on CI/CD with the master branch on both both front and backend repo, also I'll need the digital ocean webhook to be attached to both front and backend repo for me to enable it with Jenkins.
+3. I'll create a snapshot of the Taiga EC2 in Aws, and turn it down, since all traffic is on the digital ocean droplet, also should I move Taiga dB to a separate DB or leave it as it's usually the local one available in the same server as the program codes (local pg dB)????
```

Comment:
I would like it to be a separate DB for 2 reasons:
1) easier to create backup schedule

2) actualy we have a separate script that runs against it to pull cook, i was runnig it on local server but it might not in future.  its likely we will run scripts against the taiga db since it has our equity

be careful to have a fresh backup of postgres local before moving to separate db instance thx

Reply:
also, i've created a seperate DB for it thought